test_scripts/API/SetGlobalProperties/001_ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua

```diff
@@ -0,0 +1,142 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this test case when TTS.SetGlobalProperties gets WARNINGS and VR.SetGlobalProperties gets ANY successfull result code is checked
+--
+-- 1. Used preconditions:
+-- App is registered and activated SUCCESSFULLY
+-- 2. Performed steps:
+-- MOB -> SDL: sends SetGlobalProperties
+-- HMI -> SDL: TTS.SetGlobalProperties (WARNINGS), VR.SetGlobalProperties (cyclically checked cases fo result codes SUCCESS, WARNINGS, WRONG_LANGUAGE, RETRY, SAVED)
+--
+-- Expected result:
+-- SDL -> HMI: resends UI.SetGlobalProperties
```

Comment:
@istoimenova , Please see update in commit ad3b222370edd97bcd92104b132d4caa1987ed16